src/main/java/com/fasterxml/jackson/databind/deser/std/EnumDeserializer.java

```diff
@@ -76,14 +78,16 @@ public class EnumDeserializer
 
     /**
      * @since 2.9
+     * @deprecated since 2.16
      */
```

Comment:
Also usually need to add 

    @Deprecated

non-javadoc marker (Eclipse will complain otherwise). I can add these so no need to augment PR.